schema/types/conversion_mechanisms/SAFEConversionMechanism.schema.json

```diff
@@ -36,6 +36,10 @@
     "capitalization_definition": {
       "description": "How is company capitalization defined for purposes of conversion? If possible, include the legal language from the instrument.",
       "type": "string"
+    },
+    "capitalization_definition_rules": {
+      "description": "The rules for which types of securities would be included in the capitalization definition.",
+      "$ref": "https://raw.githubusercontent.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/main/schema/types/CapitalizationDefinitionRules.schema.json"
     }
   },
   "required": ["conversion_timing", "conversion_mfn", "type"],
```

Comment:
Per TWG discussion, please make the `conversion_timing` field optional. Then we're good to go.